### PR TITLE
Add text editor component for model view

### LINKED ISSUE #2057: Create text editor component for model view
- SQL Operations Studio Version:

Steps to Reproduce:

### PR DESCRIPTION
**Pull request by AbbiePetcht**
fix #2057 Create text editor component for model view
image

### CODE REVIEW

File: src/sql/sqlops.proposed.d.ts [hunk 1]
```diff
@@ -435,6 +436,11 @@ declare module 'sqlops' {
 		html?: string;
 	}
 
+	export interface EditorProperties {
```
Comment (llali) [inline src/sql/sqlops.proposed.d.ts]: Please add comment. I know not everything has comment here but we're trying to add comment to all

File: src/sql/sqlops.proposed.d.ts [hunk 2]
```diff
@@ -435,6 +436,11 @@ declare module 'sqlops' {
...
+		content?: string;
```
Comment (llali) [inline src/sql/sqlops.proposed.d.ts]: same thing for property. please add comment

File: src/sql/sqlops.proposed.d.ts [hunk 3]
```diff
@@ -503,6 +509,11 @@ declare module 'sqlops' {
 		onMessage: vscode.Event<any>;
 	}
 
+	export interface EditorComponent extends Component {
+		content: string;
```
Comment (llali) [inline src/sql/sqlops.proposed.d.ts]: comment

File: samples/sqlservices/src/controllers/mainController.ts [hunk 4]
```diff
@@ -354,15 +355,32 @@ export default class MainController implements vscode.Disposable {
...
-				], { flex: '1 1 50%' })
+			let editor1 = view.modelBuilder.editor()
+				.withProperties({
+					content: 'select * from sys.tables'
+				})
+				.component();
+
+			let editor2 = view.modelBuilder.editor()
+				.withProperties({
+					content: 'print("Hello World !")',
```
Comment (chlafreniere) [inline samples/sqlservices/src/controllers/mainController.ts]: how about a more generic default for content here?

Comment (AbbiePetcht) [reply samples/sqlservices/src/controllers/mainController.ts]: It's the extension's sample code

**Pull request merged**